### PR TITLE
More crafting (Broc and Xander)

### PR DESCRIPTION
**Pull request by roboticxlink**
!!High Priority Please!!

## Description
Adds Broc Plants!!
Adds Xander Plants!!
Adds Broc and Xander Seeds to Legion Lockers!!
Adds Broc and Xander crafting into healing powder!!

Also adds cutting up uniforms into 5 cloth.

### CODE REVIEW

**Comment by MrBobCool** [comment]
Bringing back good old drymouth crafting system one change at a time.

File: code/modules/hydroponics/grown/f13 [hunk 1]
```diff
@@ -0,0 +1,60 @@
+//fallout 13
+
```
Comment (LetterN) [inline code/modules/hydroponics/grown/f13]: what is this file for?

Comment (LetterN) [reply code/modules/hydroponics/grown/f13]: also, sync your master branch too

Comment (roboticxlink) [reply code/modules/hydroponics/grown/f13]: Gah, that file was when I was on mobile I threw the broc and xander into before putting them into their own files. And hoooowwww. I tried.

**Pull request merged**